deployment/monocular/values.yaml

```diff
@@ -14,6 +14,25 @@ sync:
     #- name: my-repo-name
     #  url: my-repository-url
     #  source: my-repository-source
+  resources: {}
```

Comment:
We need to revisit later the resources setup since monocular has proper resources settings, so we should expect to do the same with these new services. 

We can work on that in parallel in Kubeapps https://github.com/kubeapps/kubeapps/issues/478 but it would be good if you add to your TODO list to apply whatever values we came up with. Thanks.

Reply:
The current best practices are to leave resources blank rather than impose requests/limits, as usually this is best determined by the cluster operator for their environment.

Reply:
> The current best practices are to leave resources blank rather than impose requests/limits, as usually this is best determined by the cluster operator for their environment.

I do not agree with this, there is nobody better to determine the minimum required resources than the app developer. Setting limits == requests will force the scheduler to run it in QOS and not to be affected by the noise in the cluster. Being in the best interest of the developer itself.

I am ok if we do not want to enforce it, but we should at least suggest it. Although TBH in my opinion, no app should be run in production without requests/limits unless it is designed for using of residual compute.

Reply:
Sorry, I thought this was written down in the Helm docs, but I can't find it so may not actually be an agreed best practice. I do remember having a discussion over it, though, and the decision was to not set anything by default (`helm create` keeps it empty, as do most charts).

I see what you're saying, and I do think that it's useful to provide an example, but IMO it's not really possible to make a good judgement without also knowing what the available resources are and what else is running in the cluster. That may be less important for a small service like this, though.

Given that we don't have a good idea of the resource usage for chartsvc today, I think we should leave this as is and revisit it once kubeapps/kubeapps#478 is complete.

Reply:
> helm create keeps it empty, as do most charts).

That makes sense since it is the developer the one that will tweak it based on the app that it's packaging?

> I see what you're saying, and I do think that it's useful to provide an example, but IMO it's not really possible to make a good judgement without also knowing what the available resources are and what else is running in the cluster. 

Why is that? Can't we figure out what are the minimum requirements for these component? The community already defined them for the previous components https://github.com/helm/monocular/blob/master/deployment/monocular/values.yaml#L17

Note, I am talking about reasonable "minimum" requirements.

> Given that we don't have a good idea of the resource usage for chartsvc today, I think we should leave this as is and revisit it once kubeapps/kubeapps#478 is complete.

Sounds good, thanks!